apps/api/v2/src/ee/bookings/2024-08-13/services/bookings.service.ts

```diff
@@ -587,6 +588,24 @@ export class BookingsService_2024_08_13 {
     return this.outputService.getOutputReassignedBooking(reassigned);
   }
 
+  async getCalVideoSessions(bookingUid: string) {
```

Comment:
Instead of putting this in the `bookings.service.ts` can we create a new `bookings-video-sessions.service.ts` within the bookings module and it would have `getCalVideoSessions` function and then `bookings.controller.ts` will dependency inject `bookings-video-sessions.service.ts` and call `this.videoSessionsService.getCalVideoSessions`.